### ISSUE TITLE
Get the recent unfollowers / unfollowings?

### ISSUE THREAD

**Issue by PavlosTze** [bluesky-social/atproto#3064]
Hello,

I am looking into the API but couldn't find a straightforward way to get the recent unfollowers / unfollows through it. I know there are workarounds by keeping in a local storage all the current followers/follows and fetching them every time and `diff` them, but I don't think this is an elegant (nor optimal) way to do it and it will stress the resources at the end of the day.

Is there a more optimal solution in giving for example a `timestamp` or something and get back the unfollowers & unfollows since then?

**Comment by DavidBuchanan314** [comment]
Nope, you'll have to maintain your own index. Note that the "firehose" broadcasts deletion events (an unfollow is just the deletion of a follow record), which will probably help you.

**Comment by PavlosTze** [comment]
> Nope, you'll have to maintain your own index. Note that the "firehose" broadcasts deletion events (an unfollow is just the deletion of a follow record), which will probably help you.

I see, I was looking to use this functionality in a mobile app, which will be powered by the Bluesky's API so I guess it's a dead end here as I don't want to maintain my own servers etc.

**Comment by AndreiSva** [comment]
I think it is good as it is now. It would not be elegant or make sense to have a separate API option for this.
